docs/getting-started-tutorial/4-css-selector-transforms.adoc

```diff
@@ -0,0 +1,171 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+# CSS Selector Transforms
+
+Because Lift operates by transforming HTML trees, it needs an easy way to
+specify those transformations. Otherwise we'd be doing a bunch of recursive
+tree searches and munges and it would get ugly and probably slow to boot. To
+deal with transformations easily, we use a small subset of CSS selectors, with
+a few Lift idiosyncrasies to maximize performance and address some use cases
```

Comment:
Recommend the addition of a comma after the word `idiosyncrasies`. Also recommend potentially using a word other than `idiosyncrasies` as it has a negative connotation. Perhaps `Lift-specific additions`?

Reply:
Totally rephrased that bit, it was definitely a little rambly.